OpenKeychain/src/main/java/org/sufficientlysecure/keychain/KeychainApplication.java

```diff
@@ -100,12 +100,14 @@ public void onCreate() {
         // Add OpenKeychain account to Android to link contacts with keys and keyserver sync
         createAccountIfNecessary(this);
 
-        // if first time, enable keyserver and contact sync
+        // if first time, enable contact sync
         if (Preferences.getPreferences(this).isFirstTime()) {
-            KeyserverSyncAdapterService.enableKeyserverSync(this);
             ContactSyncAdapterService.enableContactsSync(this);
         }
 
+        // will set a keyserver sync if one does not exist or the interval has changed
+        KeyserverSyncAdapterService.updateKeyserverSync(this);
```

Comment:
when I disable the sync, will this change not enable it every time I start the app?